neutrons/garnet
# Logger moved to helper folder

## Description of Changes:
Small change to move logger in the helper folder

## Related Items



## To Test:
Everything should work as before


```bash
cd /path/to/my/local/garnet/repo/
git fetch origin pull/<PULL_REQUEST_NUMBER>/head:pr<PULL_REQUEST_NUMBER>
git switch pr<PULL_REQUEST_NUMBER>
conda activate <garnet_environment>
python -m pytest <item_to_test>
```


## Check list for the pull request

---

## Check list for the reviewer
